resources/certificate.rb

```diff
@@ -19,7 +19,7 @@
 # limitations under the License.
 
 property :java_home, String, default: lazy { node['java']['java_home'] }
-property :keystore_path, String, default: lazy { "#{node['java']['java_home']}/jre/lib/security/cacerts" }
+property :keystore_path, String, default: ''
```

Comment:
this makes it a breaking change

Reply:
Possibly, but I hope I am defaulting it in the code